global.json

```diff
@@ -4,6 +4,6 @@
         "Rules"
     ],
     "sdk": {
-        "version": "1.1.5"
+        "version": "2.1.4"
```

Comment:
PowerShell core sdk version isn't at 2.1.4, but only 2.0.2. I would rather not get in front of PowerShell core - https://github.com/PowerShell/PowerShell/blob/master/global.json, what do you think?

Reply:
I was originally asking the same question in the PowerShell core repo [here](https://github.com/PowerShell/PowerShell/pull/5903#issuecomment-359210325) and it seems that starting with netcore2, the sdk and runtime are decoupled now so that one can use an older sdk but a newer runtime. I could try doing it like they do in the PowerShell Core repo but at the moment, the SDK was chosen to match  the 2.0.5. runtime that it contains.

Reply:
I'm still not clear on the CI system for PSSA, does it actually do a run against Core as well as full? Can the test automation help up here?

Reply:
I'm still not clear on the CI system for PSSA, does it actually do a run against Core as well as full? Can the test automation help up here?

Reply:
Me neither but as far as I understand PSSA gets first built using `net451` only. 
It seems that the core version is built by targeting `netstandard1.6`, which should eliminate at least the runtime versioning problem. I'll add a test to make sure it builds at least fine against `netstandard1.6` in CI. I suppose running the tests using the core version and/or against pwsh are future improvements to it.

Reply:
ok -thanks-

Reply:
The only reason we decoupled the SDK and Runtime versions in PowerShell Core repo is that the new CLI versions violated PowerShell Core build process several times. I hope it doesn't matter here.